django_declarative_apis/events.py

```diff
@@ -0,0 +1,84 @@
+#
+# Copyright (c) 2025, salesforce.com, inc.
+# All rights reserved.
+# SPDX-License-Identifier: BSD-3-Clause
+# For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+#
+
+from enum import Enum
+import logging
+from django.conf import settings
+
+logger = logging.getLogger(__name__)
+
+try:
+    from newrelic import agent as newrelic_agent
+except ImportError:
+    newrelic_agent = None
+
+
+# Enum for event types
+class EventType(Enum):
+    QUEUE_LENGTH = "task_runner:queue_length"
+    TASK_RETRY = "task_runner:retry"
+
+
+def _import_hook(hook_path):
```

Comment:
should this also check that the path is a callable? an example of that can be found [here](https://github.com/salesforce/django-declarative-apis/blob/main/django_declarative_apis/models.py#L31-L36).